skin.json

```diff
@@ -33,6 +44,19 @@
     "snapwikiskin": ["i18n"]
   },
   "ResourceModules": {
+    "skins.snapwikiskin.search": {
+      "dependencies": ["mediawiki.Uri", "wvui"],
+      "packageFiles": [
+        "resources/skins.snapwikiskin.search/skins.snapwikiskin.search.js",
+        "resources/skins.snapwikiskin.search/App.vue",
+        {
+          "name": "resources/skins.snapwikiskin.search/config.json",
```

Comment:
I cannot see config.json anywhere.

Reply:
[This commit](https://github.com/wikimedia/Vector/commit/e0c47dc462b87fc1250d6e2dd6ab7020bced0c9a) explains it.